.travis.yml

```diff
@@ -1,10 +1,12 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
+  - "node"
```

Comment:
no need for `node` if all the versions are specified.  We want to be explicit when we support something new (e.g. node 7 when it comes out) but right now this just does another run of node 6.